config/default.toml

```diff
@@ -0,0 +1,5 @@
+debug = false
+port = 8000
+database_url = "mysql://root@127.0.0.1/syncstorage"
+database_use_test_transactions = false
+master_secret = ""
```

Comment:
Do we want to provide a default one or include instructions as a comment about how to do so?

Reply:
I'll add a comment. The concern with providing a default is that it actually gets used in production by accident, using the empty string makes it easy to check that hasn't happened in the code.